### ISSUE TITLE
[Feature Request] Flatpak-build

### ISSUE THREAD

**Issue by Darkitz** [Suwayomi/Tachidesk-JUI#69]
## What feature should be added to Tachidesk-JUI?
A Flatpak build would be nice.

##  Why/Project's Benefit/Existing Problem
Right now its a little bit of a pain to install this program on the Steam Deck. A proper Flatpak build would make things way easier.
(preferably even publish it on the flathub repo)

**Comment by DarkReaperBoy** [comment]
yeah, i need it since i use opensuse micro os, a rolling release immutable os

**Comment by dtantono** [comment]
we can use distrobox as a workaround. But, it does not work in gaming mode and flatpak will be nice
